src/bentoml_cli/deployment.py

```diff
@@ -127,10 +154,21 @@ def update(  # type: ignore
                 bento=bento,
                 context=shared_options.cloud_context,
                 latest_bento=True,
+                cluster_name=cluster_name,
+                kube_namespace=kube_namespace,
+            )
+        elif deployment_name is not None:
+            res = client.deployment.update(
+                deployment_name,
+                bento=bento,
+                context=shared_options.cloud_context,
+                latest_bento=True,
+                cluster_name=cluster_name,
+                kube_namespace=kube_namespace,
             )
         else:
             raise click.BadArgumentUsage(
-                "Either --file or --name is required for update command"
+                "Either --file or argument DEPLOYMENT_NAME is required for update command"
```

Comment:
You can get the DEPLOYMENT_NAME as the metavar, but it is ok to just have it here for now